java/operator/src/main/java/com/instaclustr/cassandra/operator/service/OperatorService.java

```diff
@@ -48,13 +48,19 @@ void handleDataCenterEvent(final DataCenterWatchEvent event) {
     @Subscribe
     void handleSecretEvent(final SecretWatchEvent event) {
         logger.trace("Received SecretWatchEvent {}.", event);
-        // TODO: handle updated/deleted secrets
+        // TODO: handle updated/deleted secrets - currently we don't care about this
     }
 
     @Subscribe
     void handleStatefulSetEvent(final StatefulSetWatchEvent event) {
         logger.trace("Received StatefulSetWatchEvent {}.", event);
-        // TODO
+
+        // Trigger a dc reconciliation event if changes to the stateful set has finished.
+        if(event.statefulSet.getStatus().getReplicas().equals(event.statefulSet.getStatus().getReadyReplicas()) && event.statefulSet.getStatus().getCurrentReplicas().equals(event.statefulSet.getStatus().getReplicas())) {
```

Comment:
I wonder what code style enforcement tools exist to automatically run on commit.

It'd be nice to enforce the following, for consistency:
- `final` everywhere
- whitespace after `if` and before parentheses
- always use braces, even for one-liner if statements

Reply:
Fixed some if statements. Created #106 to address